lib/kvs/kvs_rparams.c

```diff
@@ -786,6 +786,21 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "cn_incr_rspill",
+        .ps_description = "Enable incremental root spill",
```

Comment:
Why make it an option?

Reply:
Two reasons - (1) To get unit tests working in the interim (2) If we observe some behavior and would like to confirm if it's coming due to incremental spills it is easy to go back and verify that.

This rparam can be removed in the next milestone.